src/index.js

```diff
@@ -14,16 +15,19 @@ export default function (userCss, ...values) {
     css = csjs(css, ...values);
   }
 
+  let cssText = csjs.getCss(css);
+  const scope = scopedName(cssText)('DecoratedComponent');
```

Comment:
Should `scope` be something like `DecoratedComponent.name` or `scopedName(cssText)(DecoratedComponent.name)` here or is it good as-is?